qaes/encryption.py

```diff
@@ -16,10 +16,10 @@ def __init__(self, key):
         self.key = hashlib.sha256(key.encode()).digest()
         self.cipher = lambda iv: Cipher(algorithms.AES(self.key), modes.CBC(iv), backend)
 
-    def encrypt(self, plain: str) -> str:
+    def encrypt(self, plain: str, randomly=True) -> str:
```

Comment:
Since the random IV, is a regular behaviour of AES algorithms, it's preferred to use a better name for the param than `randomly`; I think `constant_iv=True` would be better.

Reply:
Fixed.